### PR TITLE
Styling changes for sphinx-tabs

### PR DESCRIPTION
**Pull request by simonbowly**
@mattmilten could you take a look at this one?

This is an attempt to get a similar look and feel to [sphinx-inline-tabs](https://sphinx-inline-tabs.readthedocs.io/en/latest/#demo) from the sphinx-tabs library we currently use. `sphinx-tabs-inline` has a much better, compact look, but we don't want to switch the underlying library since it has more features (e.g. tab sync that remembers your choice across the site).

sphinxtheme-testing is built with this branch, see here: https://gurobi-optimization-sphinxtheme-testing.readthedocs-hosted.com/en/10.0/tabs.html
Compared to the current layout: https://docs.gurobi.com/projects/optimizer/en/current/reference/misc/emptyenv.html#empty-environment-example

What do you think? One tweak I've made is the side border. I think that's needed to distinguish the content as we often use this for quite long passages of text.

### CODE REVIEW

File: gurobi_sphinxtheme/theme/static/gurobi-tabs.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    background-color: transparent;
+}
+
+.sphinx-tabs-tab:hover {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 2px solid;
+    margin: -1px;
+    border-color: var(--color-foreground-secondary);
```
Comment (mattmilten) [inline gurobi_sphinxtheme/theme/static/gurobi-tabs.css]:
```suggestion
    border-color: var(--color-brand-content);
```

File: gurobi_sphinxtheme/theme/static/gurobi-tabs.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    padding-right: 0rem;
+    border-top: none;
+    border-right: none;
+    border-bottom: none;
+    border-radius: 0;
+
+    /* Q: is the border needed? Or is the padding/indent enough? */
+    padding-left: 1rem;
+    border-left: 0.5px solid;
+    border-color: #a0b3bf;
```
Comment (mattmilten) [inline gurobi_sphinxtheme/theme/static/gurobi-tabs.css]:
```suggestion
    border-left: none;
```
I think it's fine without the left border

File: gurobi_sphinxtheme/theme/static/gurobi-tabs.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+.sphinx-tabs-tab:hover {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 2px solid;
+    margin: -1px;
+    border-color: var(--color-foreground-secondary);
+}
+body[data-theme="dark"] .sphinx-tabs-tab:hover {
+    border-color: var(--color-foreground-secondary);
```
Comment (mattmilten) [inline gurobi_sphinxtheme/theme/static/gurobi-tabs.css]:
```suggestion
    border-color: var(--color-brand-content);
```
I think this looks nicer, but I may have missed why you set this.

**Review by mattmilten — APPROVED** [review]
Pretty slick! I like it.

File: gurobi_sphinxtheme/theme/static/gurobi-tabs.css (hunk 3 above)
Comment (simonbowly) [reply gurobi_sphinxtheme/theme/static/gurobi-tabs.css]: Thanks! Was not familiar with that one, I guess I assumed it would be a garish red ...

**Comment by simonbowly** [comment]
Thanks @mattmilten! Yep, let's go without the border. I guess the idea is that the user has one language in focus throughout, so we kind of want the tabbing parts to blend into the background.

**Pull request merged**